spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplicationBuilder.java

```diff
@@ -116,15 +117,20 @@ public ConfigurableApplicationContext run(String[] parentArgs) {
 			apps.add(sinkConfigurer);
 		}
 		List<Class<?>> appsToEmbed = new ArrayList<>();
+		String namespace = null;
 		for (int i = 0; i < apps.size(); i++) {
-			appsToEmbed.add(apps.get(i).getApp());
+			AppConfigurer<?> appConfigurer = apps.get(i);
+			appsToEmbed.add(appConfigurer.getApp());
+			if (namespace == null) {
+				namespace = appConfigurer.namespace;
```

Comment:
A doubt that I had, is it desired behavior for all apps (i.e. FileSourceApplication and IngesterProcessor apps) to use a common namespace (i.e. ingester)? I'm referring to the example in #428. @dsyer could you throw some light here?